department.rb

```diff
@@ -0,0 +1,39 @@
+require './employee'
+
+class Department
+
+  attr_reader :name, :employees
+
+  def initialize(name)
+    @name = name
+    @employees = []
+    @top_performers = []
+  end
+
+  def add_employee(employee)
+    @employees << employee
+  end
+
+  def total_salaries
+    # total = 0
+    # @employees.each do |e|
+    #   total += e.salary
+    # end
+    # return total
+
+    @employees.reduce(0.0) { |sum, employee| sum += employee.salary }
+  end
+
+  def distribute_raises(budget)
+    @employees.each do |employee|
+      if employee.satisfactory?
+        @top_performers << employee
```

Comment:
You've made @top_performers an instance variable, which means that it's going to stick around between method calls.  SO, if you call distribute_raises twice, people who have been satisfactory both times will get double raises the second time!  Woot for them!

In any case, it's an extraneous use of instance variables.  Just make this local and it'll work fine.